vite.config.ts

```diff
@@ -103,6 +103,9 @@ export default defineConfig(({ mode }) => {
       environment: 'jsdom',
       globals: true,
       setupFiles: ['./src/test/setup.ts'],
+      // Performance optimizations
+      css: false, // Skip CSS processing in tests
```

Comment:
What specific speedup do we get from these particular lines?

Reply:
We don't really need CSS in JSDOM because its not gonna render so was just a speed boost

The other is source maps, we're generating source maps for tests. 

Saved about 3seconds locally

Reply:
What % speedup is 3sec for us? What run time did you see with/without?